doc/devel/spicy/autogen/tftp-no-accept.spicy

```diff
@@ -53,6 +53,9 @@ type Opcode = enum {
 type Request = unit(is_read: bool) {
     filename: bytes &until=b"\x00";
     mode: bytes &until=b"\x00";
+
+    on %done {
+    }
```

Comment:
Before reformatting spicy-tftp we would have completely omitted the `%done` hook since this was all on a single line due to https://github.com/zeek/zeek-docs/blob/2731def9159247e6da8a3191783c89683363689c/devel/spicy/autogen-spicy-docs#L42. This file seems unreferenced, and we should probably just drop that line in `autogen-spicy-docs` and remove the output file from the repo.